packages/firebase/index.d.ts

```diff
@@ -5977,11 +5978,43 @@ declare namespace firebase.database {
       value: number | string | boolean | null,
       key?: string
     ): firebase.database.Query;
+    /**
+     * Creates a `Query` with the specified ending point (exclusive).
+     *
+     * Using `startAt()`, `startAfter()`, `endBefore()`, `endAt()` and `equalTo()`
+     * allows you to choose arbitrary starting and ending points for your queries.
+     *
+     * The ending point is exclusive. If only a value is provided, children
+     * with a value less than the specified value will be included in the query.
+     * If a key is specified, then children must have a value lesss than or equal
+     * to the specified value and a a key name less than the specified key.
```

Comment:
I used "sorts before"/"sorts after" instead. I only fixed the new methods since I am not sure how to rephrase "less than or equal" to this new pattern.

Reply:
Oh that's best.  Will try to remember that phrasing!